liquibase-core/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -276,9 +278,71 @@ protected Column readColumn(CachedRow columnMetadataResultSet, Relation table, D
             }
         }
 
-        if (database.supportsAutoIncrement() && table instanceof Table) {
-            column = this.columnAutoIncrementService.enableColumnAutoIncrementIfAvailable(column, database,
-                    columnMetadataResultSet, rawCatalogName, rawSchemaName, rawTableName, rawColumnName);
+        if (database.supportsAutoIncrement()) {
```

Comment:
Hey, seems that something else was carried over this PR. This is not a try with resources merge but seems a fix from other PR that got merged here...

Reply:
Yeap, this is the result of merge conflict resolved I got when merging master into this branch.